framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -136,7 +136,8 @@ public DependentTypesHelper(AnnotatedTypeFactory factory) {
 
         this.annoToElements = new HashMap<>();
         for (Class<? extends Annotation> expressionAnno : factory.getSupportedTypeQualifiers()) {
-            List<String> elementList = getExpressionElementNames(expressionAnno);
+            List<ExecutableElement> elementList =
+                    getExpressionElementNames(expressionAnno, factory.getProcessingEnv());
```

Comment:
This is an odd name for the method if it returns ExecutableElements rather than names.